AWS/delete_iam_user.ipynb

```diff
@@ -0,0 +1,1021 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #30.        ec2Client = handle.client("iam")
</pre><p>this should be called iamClient</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/unskript/Awesome-CloudOps-Automation/pull/153/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>